packages/frontend/src/components/common/ResourceView/InfiniteResourceTableColumnName.tsx

```diff
@@ -0,0 +1,212 @@
+                multiline: true,
+                offset: -2,
+                position: 'bottom',
+            }}
+            tooltipLabel={
+                canUserManageValidation ? (
+                    <>
+                        This content is broken. Learn more about the validation
+                        error(s){' '}
+                        <Anchor
+                            component={Link}
+                            fw={600}
+                            to={{
+                                pathname: `/generalSettings/projectManagement/${projectUuid}/validator`,
+                                search: `?validationId=${validationId}`,
+                            }}
+                            color="blue.4"
+                        >
+                            here
+                        </Anchor>
+                        .
+                    </>
+                ) : (
+                    <>
+                        There's an error with this{' '}
+                        {isResourceViewItemChart(item) ? 'chart' : 'dashboard'}.
+                    </>
+                )
+            }
+        >
+            {children}
+        </ResourceIndicator>
+    );
+};
+
+type InfiniteResourceTableColumnNameProps = {
+    item: ResourceViewItem;
+    projectUuid: string;
+    canUserManageValidation: boolean;
+};
+
+const InfiniteResourceTableColumnName = ({
+    item,
+    projectUuid,
+    canUserManageValidation,
+}: InfiniteResourceTableColumnNameProps) => {
+    const isSpace = isResourceViewSpaceItem(item);
+    const isChartOrDashboard =
+        isResourceViewItemChart(item) || isResourceViewItemDashboard(item);
+
+    const hasValidationErrors =
+        isChartOrDashboard &&
+        item.data.validationErrors &&
+        item.data.validationErrors.length > 0;
+
+    const validationId = hasValidationErrors
+        ? item.data.validationErrors![0].validationId
+        : undefined;
+
+    return (
+        <Anchor
+            component={Link}
+            sx={{
+                color: 'unset',
+                ':hover': {
+                    color: 'unset',
+                    textDecoration: 'none',
+                },
+            }}
+            to={getResourceUrl(projectUuid, item)}
+            onClick={(e: React.MouseEvent<HTMLAnchorElement>) =>
+                e.stopPropagation()
+            }
+        >
+            <Group noWrap>
+                <ResourceValidationErrorIndicator
+                    item={item}
+                    projectUuid={projectUuid}
+                    canUserManageValidation={canUserManageValidation}
+                    validationId={validationId}
+                >
+                    <ResourceIcon item={item} />
+                </ResourceValidationErrorIndicator>
+
+                <Stack spacing={2}>
+                    <Group spacing="xs" noWrap>
+                        <Text
+                            fw={600}
+                            lineClamp={1}
```

Comment:
this won't display the full content but maybe I am mistaken since I don't fully understand the next line